Cargo.toml

```diff
@@ -23,14 +23,14 @@ classifier = [
 provides-extra = ["validate"]
 
 [dependencies]
-pyo3 = { version = "0.12.3", features = ["extension-module"] }
-ncollide3d = "0.26.0"
+pyo3 = { version = "0.13.2", features = ["extension-module", "abi3-py36"] }
```

Comment:
Haven't come across this before! Does it cost anything, or is it a hard "will compile/ won't compile" thing which happens to work for the subset of the abi ncollpyde uses?

Reply:
It'll fail to compile if you use non-limited APIs.